### PR TITLE
Add no-response app description

### PR DESCRIPTION
**Pull request by lee-dohm**
Let me know if there's anything else I should add :+1:

### CODE REVIEW

File: _apps/no-response.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+- https://raw.githubusercontent.com/probot/no-response/master/docs/no-response-test.png
+authors:
+- lee-dohm
+repository: probot/no-response
+host: https://probot-no-response.herokuapp.com
+---
+
+Automatically closes Issues where the author hasn't responded to a request for more information.
+
+## Bot Workflow
```
Comment (bkeepers) [inline _apps/no-response.md]: App? 😁

File: _apps/no-response.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+The intent of this bot is to close issues that have not received a response to a maintainer's request for more information. Many times issues will be filed without enough information to be properly investigated. This allows maintainers to label an issue as requiring more information from the original author. If the information is not received in a timely manner, the issue will be closed. If the original author comes back and gives more information, the label is removed and the issue is reopened if necessary.
```
Comment (bkeepers) [inline _apps/no-response.md]: > The intent of this bot…

app? 😁

File: _apps/no-response.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+* The issue has a label named the same as the `responseRequiredLabel` value in the configuration
+
+It will:
+
+1. Remove the `responseRequiredLabel`
+1. Reopen the issue if it was closed
+
+## Usage
+
+1. **[Configure the GitHub App](https://github.com/integration/probot-no-response)**
```
Comment (bkeepers) [inline _apps/no-response.md]: https://github.com/apps/no-response

File: _apps/no-response.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+# Comment to post when closing an Issue for lack of response. Set to `false` to disable
+closeComment: >
+  This issue has been automatically closed because there has been no response
+  to our request for more information from the original author. With only the
+  information that is currently in the issue, we don't have enough information
+  to take action. Please reach out if you have or find the answers we need so
+  that we can investigate further.
+```
+
+See [docs/deploy.md](docs/deploy.md) if you would like to run your own instance of this app.
```
Comment (bkeepers) [inline _apps/no-response.md]: Tests are failing because of this link.

**Comment by lee-dohm** [comment]
Updated. Thanks! 👍

File: _apps/no-response.md [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+
+The intent of this app is to close issues that have not received a response to a maintainer's request for more information. Many times issues will be filed without enough information to be properly investigated. This allows maintainers to label an issue as requiring more information from the original author. If the information is not received in a timely manner, the issue will be closed. If the original author comes back and gives more information, the label is removed and the issue is reopened if necessary.
+
+### Scheduled
+
+Once per hour, it searches for issues that are:
+
+* Open
+* Have a label named the same as the `responseRequiredLabel` value in the configuration
+* The `responseRequiredLabel` was applied more than `daysUntilClose` ago
```
Comment (j-f1) [inline _apps/no-response.md]:
```markdown
Once per hour, it searches for issues that:

* Are open
* Have had a label the named the same as the `responseRequiredLabel` value in the configuration applied more than `daysUntilClose` ago 
```

?

**Comment by j-f1** [comment]
I think you need to ⬆️ your version of `probot` so Travis can fetch the stats.

**Comment by lee-dohm** [comment]
Thanks @j-f1, I'll try that.

**Pull request merged**